fury/pick.py

```diff
@@ -73,3 +77,113 @@ def event_position(self, iren):
             using providing ShowManager's iren attribute.
         """
         return iren.GetEventPosition()
+
+    def pickable_on(self, actors):
+
+        if isinstance(actors, Sequence):
+            for a in actors:
+                a.PickableOn()
+        else:
+            actors.PickableOn()
+
+    def pickable_off(self, actors):
+
+        if isinstance(actors, Sequence):
+            for a in actors:
+                a.PickableOff()
+        else:
+            actors.PickableOff()
+
+
+class SelectionManager():
+
+    def __init__(self, select='faces'):
```

Comment:
This needs a docstring.

What happens if someone puts as a parameters `FACES` or `FACE`or `face`?